final-project/index.html

```diff
@@ -1,14 +1,352 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="./scripts/main.js"></script>
+  <script src="./scripts/faq.js"></script>
+  <script src="./scripts/blog.js"></script>
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Quicksand:300,400,500,700" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800" rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz"
+    crossorigin="anonymous">
   <link rel="stylesheet" href="./dist/styles.css">
+
 </head>
+
 <body>
-  <h1>Hello world!</h1>
-  <script src="./dist/bundle.js"></script>
+
+  <header class="header">
+    <div class="logo"></div>
+    <div class="header__nav">
+      <i class="fas fa-bars"></i>
+    </div>
+    <div class="header__menu text--one text--light text--font12 text--wide text--grey5">
```

Comment:
This would be more semantically correct if you used a `nav` and after it `ul` with `li`